src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const isMarriedMale = isMarried && sex === 'm';
+  const isMarriedFemale = isMarried && sex === 'f';
+  const isSingle = !isMarried;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+
+      {age && <p className="Person__age">I am {age}</p>}
+
+      {isMarriedMale && (
+        <p className="Person__partner">{partnerName} is my wife</p>
+      )}
+
+      {isMarriedFemale && (
+        <p className="Person__partner">{partnerName} is my husband</p>
+      )}
+
+      {isSingle && <p className="Person__partner">I am not married</p>}
```

Comment:
Need to simplify your code. Don't need to duplicate one tag with the same class three times
```suggestion
  const isMarried = !!partnerName;
  const partnerRole = sex === 'm' ? 'wife' : 'husband';

  const partnerText = isMarried
    ? `${partnerName} is my ${partnerRole}`
    : 'I am not married';

  return (
    <section className="Person">
      <h2 className="Person__name">My name is {name}</h2>

      {age && <p className="Person__age">I am {age}</p>}

      <p className="Person__partner">{partnerText}</p>
    </section>
  );

```